src/App.tsx

```diff
@@ -95,7 +132,7 @@ const App = () => {
             }}
             onContentProcessDidTerminate={() => webviewRef.current?.reload()}
```

Comment:
i do have a small concern that this will cause the webview to go back to a yat link if the app is backgrounded/comes back into "focus" and is reloaded, but needs to be tested on testflight or similar.

Reply:
shower thought - `useMemo` the "default"/dashboard route and have it go back to that instead

Reply:
fixed in https://github.com/shapeshift/mobile-app/pull/89/commits/0701ba903c895846090f5211eabb28b67648c454